src/Adapter/ContextStateManager.php

```diff
@@ -215,21 +220,29 @@ public function saveCurrentContext(): self
         return $this;
     }
 
+    /**
+     * @return array|null
```

Comment:
What does it mean when contextFieldsStack is null compared to empty array? Don't make me read the source code to understand the interface, this is what Phpdoc is for!

Reply:
Documented